### PR TITLE
Add basic Typescript definition

### PR DESCRIPTION
**Pull request by yoeran**

### CODE REVIEW

File: eev.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+declare module 'eev' {
+    type CallbackFunction = (data: any) => void
+
+    export class Eev {
+        constructor();
+        on(names: string[], fn: CallbackFunction): void;
```
Comment (chrisdavies) [inline eev.d.ts]: This should be:

`on(name: string, fn: CallbackFunction): void;`

Honestly, I don't love the way I implemented these functions. If I were to re-write Eev, I'd probably change it to be: `on(name: string | string[], fn: CallbackFunction): Eev` but alas, that's beyond the scope of this pull-request.

File: eev.d.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        off(names: string[], fn: CallbackFunction): void;
```
Comment (chrisdavies) [inline eev.d.ts]: The off signature also takes a single string, not an array.

File: eev.d.ts (hunk 1 above)
Comment (yoeran) [reply eev.d.ts]: Ah, it seems I've misread something, will fix it somewhere today. Thanks for taking the time to look at this PR.

**Comment by yoeran** [comment]
Fixed ;) I'm about to start using this library, but I needed some typings before I could use it, so I thought it won't harm if I do this thing first.

**Pull request merged**